spec/reviews_controller_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+describe  ReviewsController , :type => :controller do
+  context 'GET #index' do
+    it 'has a 200 status code' do
+      get :index
+      expect(response).to have_http_status(:success)
+    end
+    it 'returns json' do
+      get :index
+      expect(response.content_type).to eq 'application/json'
+    end
+  end
+
+  context 'POST a new review' do
+    let(:restaurant) { FactoryBot.create(:restaurant)}
+    let(:review) { FactoryBot.build_stubbed :random_review, restaurant_id: restaurant.id }
```

Comment:
Why are we using `build_stubbed`? How does it differ from using build and create?

Reply:
faster, no db access (like build)  
build_stabbed makes objects look like they’ve been persisted by creating associations.